modules/caddyhttp/reverseproxy/reverseproxy.go

```diff
@@ -208,9 +208,13 @@ func (h *Handler) Provision(ctx caddy.Context) error {
 		if err != nil {
 			return err
 		}
+
 		if addr.PortRangeSize() != 1 {
 			return fmt.Errorf("multiple addresses (upstream must map to only one address): %v", addr)
 		}
+    
+	upstream.networkAddress = addr
+
```

Comment:
```suggestion

		upstream.networkAddress = addr

```

Reply:
Sorry I didn't make it completely better last time... ran `gofmt` locally to check wtf was up.

Reply:
The annotation on Github wasn't clear where the issue was